js_modules/dagit/packages/core/src/schedules/humanCronString.test.ts

```diff
@@ -127,4 +128,36 @@ describe('humanCronString', () => {
       );
     });
   });
+
+  describe('Cron union', () => {
+    it('handles a single-item union of cron strings, with timezone', () => {
+      const timezone = 'America/Phoenix';
+      expect(humanCronString("['2,4-5 1 * * *']", timezone)).toBe(
+        'At 2 and 4 through 5 minutes past the hour, at 01:00 AM MST',
+      );
+    });
+
+    it('handles a union of cron strings', () => {
+      expect(humanCronString("['0 2 * * FRI-SAT', '0 2,8 * * MON,FRI', '*/30 9 * * SUN']")).toBe(
```

Comment:
Good tests! 

It sounds like this is actually a python serialized object string and not a JSON string? I wonder if that could be changed on the GraphQL side...